openmls/src/treesync/node/leaf_node/lifetime.rs

```diff
@@ -7,15 +7,16 @@ use tls_codec::{TlsDeserialize, TlsSerialize, TlsSize};
 /// The value is in seconds and amounts to 3 * 28 Days, i.e. about 3 months.
 const DEFAULT_KEY_PACKAGE_LIFETIME_SECONDS: u64 = 60 * 60 * 24 * 28 * 3;
 
-/// The maximum total lifetime range that is acceptable for a leaf node.
-/// The value is in seconds and amounts to 3 * 28 Days, i.e., about 3 months.
-const DEFAULT_MAX_LEAF_NODE_LIFETIME_RANGE_SECONDS: u64 = 60 * 60 * 24 * 28 * 3;
-
 /// This value is used as the default amount of time (in seconds) the lifetime
 /// of a `KeyPackage` is extended into the past to allow for skewed clocks. The
 /// value is in seconds and amounts to 1h.
 const DEFAULT_KEY_PACKAGE_LIFETIME_MARGIN_SECONDS: u64 = 60 * 60;
 
+/// The maximum total lifetime range that is acceptable for a leaf node.
```

Comment:
This is the lifetime fix mentioned in the PR description. It sets the maximum lifetime range to default + margin. One less and the tests will break (when we actually validate). I think this is what we want.